components/TextFields/src/ContainedInputView/private/MDCTextControlStyleOutlined.m

```diff
@@ -137,8 +137,13 @@ - (void)applyStyleTo:(UIView *)view
                                                               containerHeight:containerHeight
                                                                     lineWidth:outlineLineWidth
                                                               isLabelFloating:isLabelFloating];
+
+  [CATransaction begin];
```

Comment:
how come we didnt need this before and now we do?

Reply:
It was animating without me telling it to. It was getting "caught in an animation," which was leading to flakiness in the snapshot tests. For now I don't want the outlined style to animate at all.